### ISSUE TITLE
App crash

### ISSUE THREAD

**Issue by robertklep** [MarvinSchenkel/HomeyKodi#15]
Posted on Slack

![image](https://user-images.githubusercontent.com/1190916/45414794-73cfd380-b67c-11e8-86e5-47daafac5fd7.png)

Issue occurs [here](https://github.com/MarvinSchenkel/HomeyKodi/blob/222f01c80a5b9535341e95bbb8bdbe16b8990316/drivers/kodiDevice.js#L227), it looks like it can happen that `this._player` isn't defined yet when it gets called.

**Comment by MarvinSchenkel** [comment]
Yeah something weird is happening with `this._player`. I also received the following log file. Could Homey be garbage collection objects or something? Would be weird since `_kodi` is holding a reference to `_player`


```
/drivers/kodiDevice.js:217
        return this._player.pauseResume()
                            ^

TypeError: Cannot read property 'pauseResume' of undefined
    at KodiDevice.pauseResume (/drivers/kodiDevice.js:217:29)
```

**Comment by MarvinSchenkel** [comment]
Fixed in 2.2.2
